sources/Interop/Windows/um/minwinbase/PROCESS_HEAP_ENTRY.cs

```diff
@@ -24,21 +24,22 @@ public unsafe partial struct PROCESS_HEAP_ENTRY
 
         [NativeTypeName("WORD")]
         public ushort wFlags;
-        [NativeTypeName("PROCESS_HEAP_ENTRY::(anonymous union at um/minwinbase.h:242:5)")]
-        public _Anonymous_e__Union Anonymous;
+
+        [NativeTypeName("_PROCESS_HEAP_ENTRY::(anonymous union at C:/Program Files (x86)/Windows Kits/10/Include/10.0.18362.0/um/minwinbase.h:242:5)")]
+        public _Anonymous_e__Union1 Anonymous;
 
         [StructLayout(LayoutKind.Explicit)]
-        public partial struct _Anonymous_e__Union
+        public partial struct _Anonymous_e__Union1
         {
             [FieldOffset(0)]
-            [NativeTypeName("struct (anonymous struct at um/minwinbase.h:243:9)")]
-            public _Block_e__Struct Block;
+            [NativeTypeName("struct (anonymous struct at C:/Program Files (x86)/Windows Kits/10/Include/10.0.18362.0/um/minwinbase.h:243:9)")]
+            public _Anonymous_e__Union2 Block;
```

Comment:
Regression here. The struct name matches the field name, rather than `Anonymous`, when the field actually has a name.

Reply:
I created a regular expression to help me find these and corrected all occurrences.